library/patient_self_checkin/step5.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+use OpenEMR\Core\Header;
+
+$ignoreAuth = true;
+require_once "../../interface/globals.php";
+if (!$GLOBALS['self_checkin_enable'] == 1) {
+    die("This feature has not been enabled");
+}
+?>
+
+<link rel="stylesheet" type="text/css" href="/interface/themes/selfCheckIn.css">
+
+<!DOCTYPE html>
+<html>
+<head>
+
+    <title><?php echo xlt("Check In"); ?></title>
+<?php
+
+Header::setupHeader();
+
+?>
+</head>
+<body class="step5">
+
+<p>
+    <img style="text-align: center"
+         width="500px" src='<?php echo $GLOBALS['images_static_relative']; ?>/logo-full-con.png'/>
+</p>
+
+<?php
+
+$month = htmlspecialchars($_GET['month']);
+$birthday = htmlspecialchars($_GET['date']);
+$surname = htmlspecialchars($_GET['surname']);
+
+// date_default_timezone_set();
+
+$timeRN = date("Y-m-d H:i:s");
+$date = date("y-m-d");
+echo "<h1> Time: " . $timeRN . "</h1><br>";
+
+require 'mySQL_connector.php';
+
+// Get the patient IDs of everyone with an appointment today. Enter them into an array which we can call upon later.
```

Comment:
This will skip recurrent appointments. Check out how the tracker board collects appointments for the day and use that mechanism instead(note if you dig into that function, that it returns the pid of the appt):
https://github.com/bradymiller/openemr/blob/master/interface/patient_tracker/patient_tracker.php#L58